Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1

```diff
@@ -1,108 +1,194 @@
-)
-
-$url=($BaseURL.ToString().TrimEnd("/"))+'/api/v1/Indexer/schema'
-$headers=@{'X-Api-Key'=$APIKey}
-$R= Invoke-WebRequest -Uri $url -Headers $headers -ContentType 'application/json' -Method Get
-$exp_indexername= { IF ($_.IndexerUrls) {'['+$_.name+']('+($_.IndexerUrls[0])+')'+'{#'+$_.infoLink.Replace('https://wiki.servarr.com/prowlarr/supported-indexers#','')+'}'} Else {$_.name+'{#'+$_.infoLink.Replace('https://wiki.servarr.com/prowlarr/supported-indexers#','')+'}'}}
-$indexerobject= $R.Content | ConvertFrom-Json
-$tblobj= $indexerobject | Sort-Object -Property 'name'
-
-#$tbl_PubUse= $tblobj | Where-Object {($_.privacy -eq "public") -and ($_.protocol -eq "usenet")} | `
-#         Select-Object @{Name = 'Indexer'; Expression =$exp_indexername}, `
+    $AppBaseURL = 'http://localhost:9696'
+),
+
+## Gather Inputs & Variables
+### User Inputs
+#### Convert Params to match vars
+$app_baseUrl = $AppBaseURL
+$app_apikey = $AppAPIKey
+####
+
+## Start Variables
+### Application Details
+$app_api_version = 'v1'
+$app_api_path = '/api/'
+$app_api_endpoint_version = '/system/status'
+$app_api_endpoint_indexer = '/indexer/schema'
+$headers = @{'X-Api-Key' = $app_apikey }
+#### Github App Info
+$gh_app_org = 'Prowlarr'
+$gh_app_repo = 'Prowlarr'
+#$gh_app_branch = 'develop'
+## Disabled; API used for manual usage only
+### Wiki Details
+$wiki_link = 'https://wiki.servarr.com'
+$wiki_app_path = '/prowlarr'
+$wiki_page = 'supported-indexers'
+$wiki_bookmark = '#'
+### Page Formating
+$wiki_1newline = "`r`n"
+$wiki_2newline = "`r`n`r`n"
+#$wiki_encoding = 'utf8'
+### Github Details
+$gh_web = 'https://github.com'
+$gh_web_commit = 'commit/'
+#$gh_baseurl = 'https://api.github.com'
+#$gh_api_path = '/'
+#$gh_api_type = 'repos/'
+#$gh_api_branches = 'branches/'
+## Disabled; GH API used for manual usage only
+## End Variables
+Write-Host 'Variables and Inputs Imported'
+
+## Build Parameters
+### App
+$api_url = ($app_baseUrl.ToString().TrimEnd('/')) + $app_api_path + $app_api_version
+$version_url = $api_url + $app_api_endpoint_version
+$indexer_url = $api_url + $app_api_endpoint_indexer
+### Github
+$gh_repo_org = $gh_app_org + '/' + $gh_app_repo + '/'
+#$gh_url = ($gh_baseUrl.ToString().TrimEnd('/')) + $gh_api_path + $gh_api_type + $gh_repo_org + $gh_api_branches + $gh_app_branch
+## Disabled; API used for manual usage only
+## 
+### Wiki
+$wiki_infolink = ($wiki_link.ToString().TrimEnd('/')) + $wiki_app_path + '/' + $wiki_page + $wiki_bookmark
+$wiki_commiturl = ($gh_web.ToString().TrimEnd('/')) + '/' + $gh_repo_org + $gh_web_commit
+Write-Host 'Parameters Built'
+
+## Invoke Requests
+#Write-Host 'Getting Github Commit Info'
+#$github_req = Invoke-WebRequest -Uri $gh_url -ContentType 'application/json' -Method Get
+#Write-Host 'Got Github Commit Info'
+## Disabled; API used for manual usage only
+Write-Host 'Getting App Version'
+$version_req = Invoke-WebRequest -Uri $version_url -Headers $headers -ContentType 'application/json' -Method Get
```

Comment:
This and line 109 can be combined

`$version_obj = (Invoke-WebRequest -Uri $version_url -Headers $headers -ContentType 'application/json' -Method Get).Content | ConvertFrom-Json`